src/tss2-fapi/api/Fapi_Delete.c

```diff
@@ -198,15 +198,19 @@ normalize_and_check_path_list(
         if (!find_path_for_profile(context->config.profile_name, IFAPI_HS_PATH,
                                   file_ary, n))
             return_error(TSS2_FAPI_RC_BAD_PATH,
-                         "SRK has to be delted together with SRK");
+                         "SRK has to be deleted together with SRK");
```

Comment:
SRK with SRK makes no sense.
SRK with HS or HS with SRK ?

Reply:
I will also fix it in the 2.4.x branch